HadoopWindowsUserSetup.md

```diff
@@ -0,0 +1,130 @@
+****Prerequisites:****
+Before you start installing Hadoop on Windows, there are a few prerequisites that you need to have in place:
+
+1. Java Development Kit (JDK) version 8 or higher
+2. Apache Hadoop distribution suitable for Windows
+
+**Step 1:**  *Install the Java Development Kit* 
+`Hadoop is built using Java, so you’ll need to install the Java Development Kit (JDK) version 8 
+or higher on your computer. You can download the JDK from the Oracle website.
+(https://www.oracle.com/in/java/technologies/javase/javase8-archive-downloads.html) Once the download is 
+complete, run the installer and follow the instructions to install the JDK.`
+
+**Step 2:** *Download the Hadoop distribution*
+`To install Hadoop on Windows, you’ll need to download the appropriate distribution from the 
+Apache Hadoop website (https://hadoop.apache.org/releases.html). 
+You’ll want to choose the distribution that is compatible with your version of Windows (hadoop-3.3.6) and click on binary. 
+Once you’ve downloaded the distribution, extract files of hadoop-3.3.6.tar.gz and place under “C:\Hadoop”.`
+
```

Comment:
Added the detailed hadoop installation guide to windows users, I personally have followed these steps and successfully installed the latest hadoop version and configuration.